server.js

```diff
@@ -16,7 +17,15 @@ app.use("/logout", require('./routers/logout-router.js'));
 //Start server
 try{
 	app.listen(3000);
+	//database
+	let booksDB = new sqlite3.Database('.\SQL\books.db',sqlite3.OPEN_READWRITE, (err) => {
```

Comment:
do you have a .\SQL\books.db file or is this a future thing?

Reply:
I have a mainDB file after changing to a table based approach but forgot to change it in the code, so I'll update it